lib/load.php

```diff
@@ -152,6 +152,9 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.6/option.php';
 require __DIR__ . '/compat/wordpress-6.6/post.php';
 
+// WordPress 6.7 compat.
+require __DIR__ . '/compat/wordpress-6.7/formatting.php';
```

Comment:
this isn't so much about formatting as it is about CSS. perhaps a name related to CSS would clarify, such as `css-utilities.php` or `css-helpers.php`

there's a `style-engine` directory in Core I could see this going into, but so far we don't have any equivalents to what you're doing. I support something like `css-helpers.php` but I don't know if others will like that for a single function.